www/src/utils/createLink.js

```diff
@@ -44,13 +44,12 @@ const createLinkCommunity = ({item, location, section}) => {
         />
       </a>
     );
-  } else {
```

Comment:
Redundant `else`.

Reply:
I have a slight preference for subjective changes (like this) not to be included in PRs b'c they run the risk of unnecessarily churning code.

Reply:
Sure, I'll do that in future (: